### PR TITLE
feat: allow creating a new project with an avatar (#2181)

### PR DESCRIPTION
**Pull request by ciyer**
# Testing

- [x] Create a project without an avatar
- [x] Create a project with an avatar
- [x] Upload a file and then upload a new one -- the second upload should be the avatar

# Screenshots

image

/deploy #persist #cypress renku=update-renku-ui-3.1.0

### CODE REVIEW

File: client/src/project/new/components/NewProjectAvatar.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright 2021 - Swiss Data Science Center (SDSC)
```
Comment (andre-code) [inline client/src/project/new/components/NewProjectAvatar.tsx:2]: Could you please update it to 2023.

**Review by andre-code — COMMENTED** [review]
The avatar component works as expected and the code looks fine to me, I just found something to review.
 
 Could you take a look at why that comma is there and include the label about the image being cropped?

Screenshot 2023-01-30 at 10 30 05

The comma also appear in the new project form, and project setting form.
image

**Review by lorenzo-cavazzi — APPROVED** [review]
This works great! And the code looks good :+1: 
I noticed just one tiny detail. When updating an image on an already existing project, the avatar on the project detail section on the top doesn't change. Would it be easy to re-fetch it once the upload API returns a response?

_In the screenshot, notice the "uploaded" message on the bottom right and the new image, but the image on top is still the old one._

Screenshot_20230130_180252

**Review by andre-code — DISMISSED** [review]
🚀

**Comment by ciyer** [comment]
> Lgtm and tests passed 🎉
> 
> P.S. Ideally, I would keep the max size aligned between projects and datasets instead of having 3mb and 10mb. But I understand we still have the limitation from GitLab for projects, so we can live with this for now instead of optimizing the resizing function 👍
> 
> image

Yes, I think we can standardize the limit once we move to storing the avatar in the repo with renku-core.

**Pull request merged**